project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 0.13.17
\ No newline at end of file
+sbt.version = 1.2.6
```

Comment:
Notice I am bumping sbt.version.

That also meant I had to use a sbt 1.X compatible `sbt-spark-package` plugin. See below.